CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [Unreleased]
 ### Added
 - Add 'mapping_coerce' field to index resource ([#229](https://github.com/elastic/terraform-provider-elasticstack/pull/229))
+- Add 'deletion_protection' field to index resource ([#167](https://github.com/elastic/terraform-provider-elasticstack/pull/167))
```

Comment:
Can we highlight this as a breaking change, with an example of how the field should look if users want to delete an index.